cypress/e2e/course_board/openDrawingElementOnBoard.feature

```diff
@@ -1,25 +1,71 @@
 @pr
 @release
+@stable_test
+@api-migrated
 Feature: Course Board - Opening a drawing element on a course page
 
     As a teacher and student I want to open already created drawing element on a course page
 
-    @stable_test
-    Scenario Outline:  Teacher is able to open a drawing element to the course board
+    Scenario Outline: Users (admin, teacher, student), create a course with board and open TLDraw.
+        Given I am logged in as a '<admin>' at '<namespace>'
+        Given I am logged in as a '<teacher>' at '<namespace>'
+        Given I am logged in as a '<student>' at '<namespace>'
 
-        # Teacher open the drawing element to the course column
-        Given I am logged in as a 'teacher1_nbc' at 'nbc'
+        # Pre-Condition: Creating new course
+        Given I am logged in as a '<admin>' at '<namespace>'
         When I go to rooms overview
-        When I go to room 'German'
+        When I click on FAB to create a new room
+        When I click on new course create button in sub menu
+        Then I see section one area on the course create page
+        When I enter the course title '<course_name>'
+        When I select room colour as red
+        Then I select '<fullname_teacher>' from field teacher
+        Then I see substitute teacher selection box
+        Then I see button to create a course time table container
+        When I click on button Next Steps after entering the room detail in section one
+        Then I see section two area on the course create page
+        When I select '<fullname_student>' from field student
+        When I click on button Next Steps after selecting room participant details
+        Then I see the section three area as the finish page
+        When I click on button To Course Overview on the finish page
```

Comment:
After this click in the step admin is not supposed to see the course in the overview page, so pls add the following lines:

# Note: This step is not applicable for the admin user
#Then I see the course 'CypressAut Test Creation and Deletion' on the room overview page